apps/i18n/netsim/en_us.json

```diff
@@ -18,6 +18,16 @@
   "hexadecimal": "Hexadecimal",
   "infinity": "Infinity",
   "instructions": "Instructions",
+  "instructions_AddressingMessages": "<p>In this version of the simulator, you will connect to a Router instead of directly to another student.  Enter an address in the \"To\" box to send a message to another student.</p>",
+  "instructions_AutomaticDNS": "<p>In this version of the simulator a DNS service is provided for you.</p><p>Send a message in the form <pre>GET [hostname]</pre> to the DNS service to get a student's address.</p>",
+  "instructions_default": "<p>You have loaded NetSim in superuser mode.</p>",
+  "instructions_SendAB": "<p><i>UNDER CONSTRUCTION: This version of NetSim is not yet functional.</i></p><p>It will allow students to manually set/read individual As and Bs on a simplex wire.</p>",
+  "instructions_SendAscii": "<p><i>UNDER CONSTRUCTION: This version of NetSim is not yet functional.</i></p><p>It will allow students to send ASCII to each other.</p>",
+  "instructions_SendBits": "<p><i>UNDER CONSTRUCTION: This version of NetSim is not yet functional.</i></p><p>It will allow students to send binary as zeroes and ones to one another</p>",
+  "instructions_ManualDNS": "<p>In this version of the simulator you can't see the other students' addresses.  Instead, one of you must act as a name service, to help share addresses with each other.",
+  "instructions_variant1": "<p>You have loaded the NetSim variant 1 template.</p><p><i>UNDER CONSTRUCTION: Variant 1 is not yet functional because P2P connections have not been implemented.</i></p>",
+  "instructions_variant2": "<p>You have loaded the NetSim variant 2 template.</p><p><i>UNDER CONSTRUCTION: Variant 2 is not yet functional because P2P connections have not been implemented.</i></p>",
+  "instructions_variant3": "<p>You have loaded the NetSim variant 3 template.</p>",
```

Comment:
I suspect this is a big fat security hole and I shouldn't do it.  Is there a nicer path for allowing formatting in i18n strings?